types/linkifyjs/linkifyjs-tests.tsx

```diff
@@ -68,6 +68,19 @@ describe("linkifyjs/html", () => {
         }
     });
 
+    linkifyHtml('', {
+        format(value, type) {
+            if (type === 'url') {
+                return <span>{value}</span>;
+            }
+            return (
+                <span>
+                    {value} ({type})
+                </span>
+            );
+        },
+    });
```

Comment:
This doesn't seem to test that `format()` can return a string.

Reply:
There's an existing `format` test above that returns only strings, so I had this one returning only elements. I can also add a test where `format` returns strings and elements.

Reply:
Oh, I totally forgot that `value` is a string -- that's why I didn't think any of the functions above were returning strings. Thanks for explaining!